domains/kucharskov.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Kucharskov",
+           "email": "",
+           "discord": "271314313984212992",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.B4bLEFARNAdE_BpPAqvnVufPoWEFTizcv4TOGjyRp4RXYC2M5KDV9FYxsJGHw1FSL349cL58sYIEj9rGNhYxAPWf9TzEXKgnCCHPcjbnOFr4q4Vo1z29HuvY2DyYjKHpWuGDoIgaYNsnXz2vYjgLeHG8gc6tB6v53Urf9jj4tF_vE3DmxgdsuXNcNlh9Lmw9Xi1DA2dId5YGklN9MVJLL78LEWkADBidKsSeQkEQ3h98pPw6MQuZ1WcDCBiFs6SaL0Hp9Mjsn3XESdwUeSF8oFbuIyRqQOsr26-SSn8t4noKeI_yvzI36sItkJVkfho9QtYUcdfjcxerXNDSpJ-6ow.fmZCDBax-e6Fvc2CpmcvUQ.TYO6CzoUcOp_GxVs9ACfSWNB4tawICrusIk8V1ccw4_4LrT5Sg8TSqX6Hv9AkSzHa1uJjqnq8Fpe5kLW7jp5neM1eC8YpeWKkpr_bxyggGA.OR3Hp4ja27cqEwLpfgmFag"
+        },
+    
+        "record": {
+            "A": ["89.117.94.211"]
```

Comment:
Can you show a preview?

Reply:
At this time i got there nginx default page but it will be redirected (on the nginx) side to my website https://kucharskov.pl
![Screenshot_20240919_144738_Chrome](https://github.com/user-attachments/assets/3853ffaf-ce5c-4cbf-bfcf-4cf19b18e21c)

Reply:
Is it enogh? I dont want to change my default website. When is-a.dev will be working then i will be comfortable with serving dev things